rest-api-cloud-client/src/main/java/com/stnetix/ariaddna/externalcloudapi/implementation/YandexDiskHelper.java

```diff
@@ -0,0 +1,194 @@
+package com.stnetix.ariaddna.externalcloudapi.implementation;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.stnetix.ariaddna.externalcloudapi.MediaTypes;
+import okhttp3.*;
+
+import java.io.*;
+
+public class YandexDiskHelper {
+
+    private static JsonParser parser;
+
+    static final String APP_ROOT = "app:/";
+
+    static Request baseRequest;
+
+    static final HttpUrl DISK_PATH;
+
+    static final HttpUrl RESOURCES_PATH;
+
+    static final HttpUrl COPY_PATH;
+
+    static final HttpUrl MOVE_PATH;
+
+    static final HttpUrl DLOAD_PATH;
+
+    static final HttpUrl ULOAD_PATH;
+
+    static final HttpUrl FILES_PATH;
+
+    static final HttpUrl VCODE_REQ_PATH;
+
+    static final HttpUrl TOKEN_REQ_PATH;
+
+    static final String CLIENT_ID = "141588389ce24e4e80e6ccd5db81c7a6";
+
+    static final String CLIENT_SECRET = "7fbc2f7214b64d53809249b5534291d2";
+
+    static {
+        parser = new JsonParser();
+
+        DISK_PATH = new HttpUrl.Builder()
+                .scheme("https")
+                .host("cloud-api.yandex.net")
```

Comment:
If version api is change we should be recompile software? )))
Please use DB or System Config for all path parameters